engine/access/state_stream/backend/backend.go

```diff
@@ -33,11 +35,14 @@ const (
 	// DefaultResponseLimit is default max responses per second allowed on a stream. After exceeding
 	// the limit, the stream is paused until more capacity is available.
 	DefaultResponseLimit = float64(0)
+
+	// DefaultHeartbeatInterval specifies the block interval at which heartbeat messages should be sent.
+	DefaultHeartbeatInterval = 1
```

Comment:
I think default values should be moved to `state_stream` package, it will make imports cleaner in client code.